MIGRATIONS.md

```diff
@@ -3,6 +3,12 @@
 This file documents changes in the data model. Please explain any changes to the
 data model as well as any custom migrations.
 
+## WordPress 150
+
+@momozw 2023-06-20
+
+- `Blog`: removed `isBlazeApproved` attribute.
```

Comment:
It may be worth deprecating it instead of removing to avoid the database migration.

Reply:
I'd prefer to remove isBlazeApproved all together, unless there's a clear benefit to deprecating / avoiding a lightweight migration. Wdyt @kean @hassaanelgarem?

Reply:
Technically for lightweight migration I don't think we even need to do model versioning. On WPiOS we've been creating a new version for every change, but DayOneiOS doesn't. (Ref: p1666793197893439-slack-C02HT04NTU3)

> if there are as fewer versions as possible

True. I've started a convo on Slack about dropping model versioning for changes compatible with lightweight migrations (Ref: p1687364482569129-slack-C04U3GCAVK2)